### ISSUE TITLE
Followed README but no data showing for my chain

### ISSUE THREAD

**Issue by njmurarka** [ping-pub/explorer#149]
I am trying to run my own instance of the explorer.

I configured it all according to the docs, unless I missed something.

First of all, I do not have a mainnet. So if I do not put a config in src/chains/mainnet, it seems my network does not even show up. So what do I do here -- put in a bogus mainnet entry?

Secondly, for the testnet version, I am not seeing any data showing for my chain. All empty. 

Here is my bluzelle.json file in src/chains/testnet:

```
{
    "chain_name": "bluzelle",
    "api": ["http://localhost:1327"], 
    "rpc": ["http://localhost:26667"],
    "sdk_version": "0.42.6",
    "coin_type": "483",
    "min_tx_fee": "800",
    "assets": [{
        "base": "ubnt",
        "symbol": "BNT",
        "exponent": "6",
        "coingecko_id": "bluzelle", 
        "logo": "https://dl.airtable.com/.attachments/e54f814bba8c0f9af8a3056020210de0/2d1155fb/cosmos-hub.svg"
    }],
    "addr_prefix": "bluzelle",
    "logo": "https://dl.airtable.com/.attachments/e54f814bba8c0f9af8a3056020210de0/2d1155fb/cosmos-hub.svg"
}
```

Is there some way to see the logs of why this is not working?

**Comment by njmurarka** [comment]
Yep, shows up if I put a config file in the mainnet directory. So I need to have an entry there even if I have no mainnet?

Which config file is used when I goto the UI in the browser then? How do I switch over to see the testnet and not the mainnet?

Thanks.

**Comment by liangping** [comment]
You need set a domain starts with `testnet.*.*`

**Comment by njmurarka** [comment]
Sorry, but I do not understand. 

Set the domain for what part, to start with testnet.*.*? Are you referring to the API endpoint for my testnet chain has to start with testnet? As in, testnet.bar.com? Is that a requirement for a testnet to appear? 

Also, if I do that, how do I actually navigate in the web app to see the testnet then? I do not see a button anywhere to view testnet.

**Comment by liangping** [comment]
`testnet.abc.com` will switch to testnet mode where configs are loaded from testnet directory

**Comment by liangping** [comment]
> Are you saying PingPub will ONLY display the testnet info, if the testnet-prefixed domain is provided in the config? I am still unclear here, as there is no way in PingPub to select if I want to see mainnet or testnet -- how do I select?

yeah. `testnet` domain enable configurations for testnet
